test/functional/api/v1/profiles_controller_test.rb

```diff
@@ -30,6 +34,9 @@ def response_body
         end
 
         should respond_with :success
+        should "not return owner mfa information by default" do
+          refute_match "disabled", @response.body
```

Comment:
I think we can add also key name here (and also on all other places). Maybe refactor that to methods like `assert_mfa_info_included` and `refute_mfa_info_included` to not repeat it everywhere?